### PR TITLE
SQL, Controllers, init web interface

### PR DESCRIPTION
**Pull request by DanaKhromenko**
In this PR:
- Updated models
- Added DataProcessingException
- Removed redirecting to 8082 port
- Added init_db.sql file with code to set up a local database
- Updated pom.xml
- Added MySQL implementation for Tool Dao
- Added ConnectionUtil with getConnection() method and dummy credentials
- Removed DummyController.java
- Added web app data
- Added Index and Tool Controllers

Temporary interface:
interface

### CODE REVIEW

File: src/main/resources/init_db.sql
Comment (VadimKhromenko) [inline src/main/resources/init_db.sql:1]: This file will save us a lot of time!

File: src/main/webapp/WEB-INF/css/table_light.css
Comment (VadimKhromenko) [inline src/main/webapp/WEB-INF/css/table_light.css:1]: I will start front part at the next week.

File: src/main/webapp/WEB-INF/views/index.jsp [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+<body>
+<form method="post" id="redirect"></form>
+<h1 class="table_light">Welcome to TECHNICIANS!</h1>
+<table class="table_light">
+    <tr>
+        <th>Redirect to</th>
+    </tr>
+    <tr><td><a href="${pageContext.request.contextPath}/technicians/">Display All Technicians</a></td></tr>
+    <tr><td><a href="${pageContext.request.contextPath}/TBA">Sign In</a></td></tr>
+    <tr><td><a href="${pageContext.request.contextPath}/TBA">Sign Up</a></td></tr>
```
Comment (VadimKhromenko) [inline src/main/webapp/WEB-INF/views/index.jsp:18]: Temporary solution?

**Review by VadimKhromenko — APPROVED** [review]

File: src/main/webapp/WEB-INF/css/table_light.css
Comment (DanaKhromenko) [reply src/main/webapp/WEB-INF/css/table_light.css:1]: Awesome!

File: src/main/webapp/WEB-INF/views/index.jsp (hunk 1 above)
Comment (DanaKhromenko) [reply src/main/webapp/WEB-INF/views/index.jsp:18]: For sure! Just temporary dummy data.

**Pull request merged**